.github/workflows/xep-validation.yml

```
@@ -24,10 +24,12 @@ jobs:
 
       - name: Validate changed file(s)
         if: steps.changed-xeps.outputs.any_changed == 'true'
+        env:
+          ALL_CHANGED_FILES: ${{ steps.changed-xeps.outputs.all_changed_files }}
         run: |
           sudo apt-get install -y libxml2-utils
           result=0
-          for xep in "${{ steps.changed-xeps.outputs.all_changed_files }}"; do
+          for xep in ${ALL_CHANGED_FILES}; do
```

I believe you could make this change simpler just by removing the quotes here (untested):
```suggestion
          for xep in ${{ steps.changed-xeps.outputs.all_changed_files }}; do
```

---

... possibly, but I definitely tested the commit as-is, and I don't think the proposed modification makes things considerably more easy. I'm happy to go with the commit as-is, unless someone feels strongly about this and wants to test the alternative.